examples/cli/open-page/README.md

```diff
@@ -10,5 +10,5 @@ as the argument to the `open-page` option.
 
 ## What Should Happen
 
-The script should open `http://localhost:8080/` in your default browser.
-You should see the text on the page itself change to read `Success!`.
+The script should open `http://localhost:8080/example.html#page1` in your
+default browser. You should see the text on the page itself change to read `Success!`.
```

Comment:
I think better create new example, no need modify current

Reply:
I did create a new example. I just spotted a typo in yours and fixed it?

Reply:
oh, i see, thanks! All good